test/Upgrade.ts

```diff
@@ -43,16 +47,24 @@ describe("Upgrade", () => {
   before(async () => {
     [deployer, reserve, managingBoard, user1, user2, user3] =
       await ethers.getSigners();
-    ({ token, registry, resolution } = await deployDAO(
+    const n = await NeokingdomDAOMemory.initialize({
```

Comment:
could you rename `n` to something meaningful like `daoDeployer`, `neoKingdomDAOMemory`, `neoKingdomDAO`, `daoDeployment`, `neoKingdomDAODeployment`.... ?